doc/Doxyfile_common

```diff
@@ -33,12 +33,9 @@ DOT_IMAGE_FORMAT = svg
 PREDEFINED = __GNUC__ \
 	     __attribute__(x)= \
 	     ODP_ALIGNED(x)= \
-	     ODPDRV_ALIGNED(x)= \
 	     __BIG_ENDIAN_BITFIELD \
 	     __LITTLE_ENDIAN_BITFIELD \
 	     __x86_64__ \
 	     ODP_PACKED \
-	     ODPDRV_PACKED \
 	     ODP_DEPRECATE(x)=x \
-	     "ODP_HANDLE_T(type)=odp_handle_t type" \
-	     "ODPDRV_HANDLE_T(type)=odpdrv_handle_t type"
+	     "ODP_HANDLE_T(type)=odp_handle_t type"
```

Comment:
@muvarov could you please port this part for master branch also?

Reply:
@lumag ok  I will stage to next branch changes.

Reply:
@lumag  actually that changes are not in master. So I will just not take them to next branch. So this chunk is only api-next specific.